modules/installation-configuration-parameters.adoc

```diff
@@ -190,6 +193,15 @@ endif::[]
 ifeval::["{context}" == "installing-restricted-networks-vmc"]
 :vmc:
 endif::[]
+ifeval::["{context}" == "installing-vmc-user-infra"]
+:vmc:
+endif::[]
+ifeval::["{context}" == "installing-vmc-network-customizations-user-infra"]
+:vmc:
+endif::[]
+ifeval::["{context}" == "installing-restricted-networks-vmc-user-infra"]
+:vmc:
+endif::[]
```

Comment:
I don't see the corresponding `:!vmc:` at the end of this file. Can you make sure that these attributes get un-set at the end of the file?